uberfire-extensions/uberfire-layout-editor/uberfire-layout-editor-client/src/main/java/org/uberfire/ext/layout/editor/client/LayoutEditorView.html

```diff
@@ -1,25 +1,35 @@
-<div class="le-default">
+<div data-field="mainDiv">
+    <div data-field="tabsDiv">
+        <ul class="nav nav-tabs nav-tabs-pf">
+            <li data-field="designTab" class="active" style="margin-left: 10px"><a href="#" data-field="designAnchor">Editor!</a></li>
+            <li data-field="previewTab"><a href="#" data-field="previewAnchor">Preview!</a></li>
```

Comment:
i18n?

Reply:
If there is no preview, we will have the design tab right? This is ok for forms? /cc @pefernan

Reply:
@ederign I'm using use GWT i18n to inject constants from the view into the html.
The tabs are disabled by default. They are only explicitly enabled in perspective editor.